recipes/abess/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "abess" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4b3d615efad34f0a24255594b27cec9ff2aae41d2a70812308c4bcceb02b45bb
+
+build:
+  number: 0
+  skip: True  # [py<35 and win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build: 
+    - {{ compiler('cxx') }}
+    - python
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - pip
+    - python
+    - pybind11 >=2.6.0
+    - numpy
+  run:
+    - {{ pin_compatible('numpy') }}
+    - scipy 
+    - scikit-learn >=0.24
+    - _openmp_mutex   # [linux]
+    - libstdcxx-ng    # [linux]
+    - libgcc-ng       # [linux]
+    - libcxx          # [osx]
+    - python
+
+test:
+  imports:
+    - abess
+  requires:
+    - pytest
```

```suggestion
test:
  imports:
    - abess
  requires:
    - pytest
    - pip
  commands:
    - pip check
```

Let's also add a pip check here.